packages/adblocker-webextension/adblocker.ts

```diff
@@ -402,10 +402,12 @@ export class WebExtensionBlocker extends FiltersEngine {
           scripts,
           styles: '',
         });
+        return;
       }
     }
 
     await Promise.all(promises);
+    sendResponse();
```

Comment:
Note that there are other cases above in the code that short-circuit the execution of the function with `return;` without calling `sendResponse`. Would it make sense to also call `sendResponse` on all of them?